samples/valueentity-counter/first-service.iml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
intellij file. Better to exclude with .gitignore

Reply:
https://github.com/lightbend/akkaserverless-codegen/issues/130